wgpu-native/src/device.rs

```diff
@@ -2018,6 +2050,8 @@ pub fn device_create_swap_chain<B: GfxBackend>(
                 value: device_id,
                 ref_count: device.life_guard.ref_count.clone(),
             },
+            // TODO: what should be here? UNINITIALIZED or OUTPUT_ATTACHMENT? Or both?
+            usage: resource::TextureUsage::UNINITIALIZED,
```

Comment:
let's have `OUTPUT_ATTACHMENT` only

Reply:
Also in the tracker a few lines below?

Reply:
right, the tracker should still consider it UNITIALIZED